doc/webui/README.md

```diff
@@ -24,46 +26,233 @@ single instances can be configured independently.
+
++ Response 200 (application/json)
+
+        {
+            "id": "507f191e810c19729de860ea",
+            "name": "test instance",
+            "host": "192.168.0.5"
+        }
+
+##### edit a single instance [PUT]
+
++ Request update host of instance (application/json)
+
+        {
+            "host": "192.168.0.6"
+        }
+
++ Response 204
+
+##### delete a single instance [DELETE]
+
++ Response 204
+
+##### get version of a single instance [GET /instances/{instance_id}/version]
+
++ Response 200 (application/json)
+
+        {
+            "api": 1,
+            "elektra": "0.8.17"
+        }
+
+
+
+
+#### /instances/{instance_id}/kdb/{path}
+
+you can access a single instances' configuration via `clusterd` the same way you would directly access it
+
++ Parameters
+    + instance_id: `507f191e810c19729de860ea` (string) - id of an instance
+    + path: `user/hello` (string) - path to the elektra config
+
+##### get configuration [GET]
+
+this is the same as calling `kdb get {path}` on the instance
+
++ Response 200 (application/json)
+
+        "hello world"
+
++ Request nonexistant path
+    + Parameters
+        + path: `user/nonexistant`
+
++ Response 404
+
+##### set configuration [POST]
+
+this is the same as calling `kdb set {path}` on the instance
+
++ Request (application/json)
+
+        "hello world"
+
++ Response 204
+
+##### delete configuration [DELETE]
+
+this is the same as calling `kdb rm {path}` on the instance
+
++ Response 204
 
-The cluster management server (`clusterd`) also provides a RESTful HTTP API.
-Single instances can be configured as follows:
++ Request nonexistant path
+    + Parameters
+        + path: `user/nonexistant`
 
- * **GET /instances** - get a list of all instances
- * **POST /instances** - create a new instance
- * **GET /instances/:id** - get information about a single instance
- * **PUT /instances/:id** - edit a single instance
- * **DELETE /instances/:id** - delete a single instance
- * **GET /instances/:id/kdb** - get full configuration of an instance
- * **GET /instances/:id/kdb/:path** - get `path` configuration of an instance (same as `kdb get path`)
- * **POST /instances/:id/kdb/:path** - edit `path` configuration of an instance (same as `kdb set path`)
- * **GET /instances/:id/version** - get `elektrad` version of an instance
++ Response 404
 
-It is also possible to create and manage groups of multiple elektra instances (clusters). The API is the same as above, but with `/clusters` instead of `/instances`. Additionally, you can get the `clusterd` version:
 
- * **GET /version** - get `clusterd` version
 
-Cluster configuration is stored on the cluster management server and persisted
-to all instances.
 
-### Response codes
+#### /clusters
 
- * 200: success
- * 404: non-existing key
- * 403: authorization failure
+It is also possible to create and manage groups of multiple elektra instances (clusters).
+The API is the same as above, but with `/clusters` instead of `/instances`.
```

Comment:
You'll obivously need to write this formally, plain text won't work here anymore.